tests/test_move_to_top_or_bottom_of_range.py

```diff
@@ -0,0 +1,19 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pages.treeherder import TreeherderPage
+
+
+def test_move_to_top_of_range(base_url, selenium):
+    page = TreeherderPage(base_url, selenium).open()
+    result_set = page.ResultSet(page)
```

Comment:
I can see why you did this, but it's unnecessary. You can see in this test that after opening the page, the actions can be taken by going to page.pageRegionName.method() [example: https://github.com/mozilla/treeherder-tests/blob/master/tests/test_add_new_jobs.py#L23].